web/src/components/feature/PhaseTable.tsx

```diff
@@ -0,0 +1,45 @@
+import { DynamicTable, TDynamicTableInput } from '../common/DynamicTable'
+import { MultiflashResponse } from '../../api/generated'
+
+function transformRawResponse(
+  multiFlashResponse: MultiflashResponse,
+  cubicFeedFlow: number
+): TDynamicTableInput {
+  const tableInput: TDynamicTableInput = {
+    Phases: [],
+    [`Mass Concentration`]: [],
+    [`Mole Concentration`]: [],
+    [`Mercury Flow (g/d)`]: [],
+  }
+  const phasesInResult: string[] = Object.keys(multiFlashResponse.phaseValues)
+  phasesInResult.forEach((phase, index) => {
+    tableInput['Phases'].push(phase)
+    tableInput['Mass Concentration'].push(
+      multiFlashResponse.phaseValues[phase]['mercury'].toString()
+    )
+    tableInput['Mole Concentration'].push(
+      multiFlashResponse.componentFractions['5'][index].toString()
```

Comment:
I assume the '5' is the id for mercury.
But coming back after 2 years, that would not be obvious. A small comment would help.